src/NzbDrone.Core.Test/Download/DownloadClientTests/FreeboxDownloadTests/TorrentFreeboxDownloadFixture.cs

```diff
@@ -0,0 +1,371 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using FluentAssertions;
+using Moq;
+using NUnit.Framework;
+using NzbDrone.Common.Disk;
+using NzbDrone.Common.Http;
+using NzbDrone.Core.Download;
+using NzbDrone.Core.Download.Clients;
+using NzbDrone.Core.Download.Clients.FreeboxDownload;
+using NzbDrone.Core.Download.Clients.FreeboxDownload.Responses;
+using NzbDrone.Core.Parser.Model;
+
+namespace NzbDrone.Core.Test.Download.DownloadClientTests.FreeboxDownloadTests
+{
+    [TestFixture]
+    public class TorrentFreeboxDownloadFixture : DownloadClientFixtureBase<TorrentFreeboxDownload>
```

Comment:
This will need to be updated to match the subject class name as well.

Reply:
Depends on decision made here https://github.com/Sonarr/Sonarr/pull/5188#discussion_r1050578102